zum-front-end/src/lib/zum-store.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Title: ZUM Store
+ * Description: It stores data in the browser localStorage and makes global variables.
+ * Author : Seokhyeon Jang (coolman555@me.com)
+ */
+
+export default class zumStore {
+  public commit;
+
+  constructor(props) {
+    const { state, mutations } = props;
+    if (!localStorage.getItem("zumStore")) {
+      this.state = state;
+    }
+    this.commit = mutations;
+  }
+
+  get state() {
+    let data = localStorage.getItem("zumStore");
+    if (data) {
+      return JSON.parse(JSON.parse(data))
+    }
+  }
+
+  set state(newState) {
+    const temp = JSON.stringify({ ...this.state, ...newState });
+    localStorage.setItem("zumStore", JSON.stringify(temp));
```

Comment:
```suggestion
    localStorage.setItem("zumStore", JSON.stringify({ ...this.state, ...newState }));
```
이렇게 stringify는 한 번만 사용해도 좋을 것 같습니다.